packages/docs-markdown/src/test/suite/helper/getHeader.test.ts

```diff
@@ -7,12 +7,9 @@ const expect = chai.expect;
 
 suite('getHeadings', () => {
 	test('Get heading', async () => {
-		const filePath = resolve(
-			__dirname,
-			'../../../../../src/test/data/repo/articles/docs-markdown.md'
-		);
+		const filePath = resolve(__dirname, '../../../../../src/test/data/repo/articles/bookmark.md');
 		const content = readFileSync(filePath, 'utf8');
 		const heading = getHeadings(content);
-		expect(heading).to.equal('Docs Markdown Reference');
+		expect(heading[0]).to.equal('# This is a bookmark page');
```

Comment:
Could we had more bookmark tests, to ensure that multiple H{n} values work? Also, I thought that hand made bookmarks used to work too?

For example:

```md
<a name="custom-bookmark-name"></a>
```

It would find "custom-bookmark-name" too, is that not the case anymore?

Reply:
Seems like that would be good to find custom bookmarks, it wouldnt be too hard to add this back if you just made a function to find all anchor tags with name attribute.

Reply:
I looked into the custom/handmade bookmark link and it doesn't look like we supported that type of link. I tested against a few of the earlier versions of the extension and none of them supported it. 

We can definitely roll in a few more tests.

Reply:
Created custom bookmark [task ](https://ceapex.visualstudio.com/Engineering/_boards/board/t/AXE%20Team/Stories/?workitem=435760)for tracking and requirements.